apispec/openapi-circe/src/main/scala/sttp/tapir/openapi/circe/package.scala

```diff
@@ -75,7 +75,7 @@ trait TapirOpenAPICirceEncoders {
   implicit val encoderInfo: Encoder[Info] = deriveEncoder[Info]
   implicit val encoderContact: Encoder[Contact] = deriveEncoder[Contact]
   implicit val encoderLicense: Encoder[License] = deriveEncoder[License]
-  implicit val encoderOpenAPI: Encoder[OpenAPI] = deriveEncoder[OpenAPI]
+  implicit val encoderOpenAPI: Encoder[OpenAPI] = deriveEncoder[OpenAPI].mapJsonObject(json => json.filter{ case (_, v) => !v.isNull}.mapValues(_.deepDropNullValues))
```

Comment:
do we need the `filter` - isn't `deepDropNullValues` enough?

Reply:
yep, without that filter, only nested nulls were dropped but the ones within first braces stayed

Reply:
ok, it can be simplified to `deriveEncoder[OpenAPI].mapJson(json => json.deepDropNullValues)` i will make correction